### PR TITLE
New Task Improve the UI of game

### PR DESCRIPTION
**Pull request by YashKumarVerma**
---

# Labyrinth: Build/Inprove the UI of the game

The [FOSSASIA Labyrinth](https://github.com/fossasia/labyrinth/) allows you to contribute parts to a huge labyrinth.
To extent the labyrinth, please add a new level.

Requirements:
- Add the basic UI elements (zoom in, zoom out, pan etc, reset, etc) to the game / Improve( in terms on compatibility, ease of access, solve bugs) if they already exist.
- Develop / Enhance the on-screen controls.

Expected Outcome:
- A systematic (uniform) UI elements that match the overall design of the game.

Hints:
- If someone wants to work with you, let them do that. You can use techniques like [pair programming](https://www.youtube.com/watch?v=vgkahOzFH2Q) or collaborate together on branches of your forks. You can attribute a pull-request also to other people who helped you out or submit a pull-request where you and an other person contributed together.
- You can join our [chat on gitter](https://gitter.im/fossasia/labyrinth) if you get stuck to get help or just hang out and have a nice time with others.

---

Categories:
- [X] Code
- [X] User Interface
- [ ] Documentation / Training
- [x] Quality Assurance
- [ ] Outreach / Research

### CODE REVIEW

**Comment by YashKumarVerma** [comment]
@niccokunzmann

File: tasks/inprove-the-UI-of-the-game.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<!-- https://codein.withgoogle.com/dashboard/tasks/4946505678979072/ -->
+
+---
```
Comment (abishekvashok) [inline tasks/inprove-the-UI-of-the-game.md]: is this required?

Comment (YashKumarVerma) [reply tasks/inprove-the-UI-of-the-game.md]: i don't currently have the link. so just commented it out from the sample.

Comment (abishekvashok) [reply tasks/inprove-the-UI-of-the-game.md]: Not the link the `---`

Comment (niccokunzmann) [reply tasks/inprove-the-UI-of-the-game.md]: The --- is the divider I added so I know what to copy. Its nice to have.

File: tasks/inprove-the-UI-of-the-game.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+# Labyrinth: Build/Inprove the UI of the game
+
+The [FOSSASIA Labyrinth](https://github.com/fossasia/labyrinth/) allows you to contribute parts to a huge labyrinth.
+To extent the labyrinth, please add a new level.
```
Comment (niccokunzmann) [inline tasks/inprove-the-UI-of-the-game.md]: We can change this line, too.

**Comment by niccokunzmann** [comment]
Thanks!

File: tasks/inprove-the-UI-of-the-game.md (hunk 1 above)
Comment (abishekvashok) [reply tasks/inprove-the-UI-of-the-game.md]: oh am sorry

**Pull request merged**